onnxscript/ir/_core_test.py

```diff
@@ -847,6 +847,24 @@ def test_successors(self):
     def test_successors_are_unique(self):
         self.assertEqual(self.node.successors(), (self.node_a, self.node_b))
 
+    def test_serialize_node(self):
```

Comment:
Could you add the tests to https://github.com/Johansmm/onnxscript/blob/custom-name-in-function/onnxscript/ir/serde_test.py instead? Thanks!